extensions/spring-di/deployment/src/main/java/io/quarkus/spring/di/deployment/SpringDiBuildTimeConfig.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.spring.di.deployment;
+
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+@ConfigMapping(prefix = "quarkus.spring-di")
+@ConfigRoot(phase = ConfigPhase.BUILD_TIME)
+public interface SpringDiBuildTimeConfig {
+
+    /**
+     * Whether Spring DI is enabled **during the build**.
+     * <p>
+     * Turning this setting off will result in Quarkus completely ignoring beans annotated with Spring annotations
```

Comment:
```suggestion
     * Turning this setting off will result in Quarkus completely ignoring beans annotated with Spring annotations to reduce possible side-effects when Spring DI is brought in transitively (e.g. via quarkus-spring-data-jpa).
```
Maybe?

Reply:
I don't think we need to be specific to be honest

Reply:
Alright, I was a bit on the fence myself.